src/python/WMCore/JobSplitting/JobFactory.py

```diff
@@ -200,9 +202,24 @@ def commit(self):
 
                     job['possiblePSN'] = locSet
                     if len(job['possiblePSN']) == 0:
+                        # NOTE: If we have no place to execute a single job we mark it as failedOnCreation.
+                        #       We have two options:
+                        #       * Either propagate the job state to 'createdfailed' here immediately
+                        #       * Or we let JobCreatorPoller later catch the marker and do it for us
+                        #         when moving the jobs from state 'new' to 'created'
+                        #       For the time being we opt for the later
+                        job["failedOnCreation"] = True
+                        job["failedErrCode"] = 71101
+                        job["failedReason"] = WM_JOB_ERROR_CODES[job["failedErrCode"]]
+                        msg = "Found bad job: %s from JobGroup: %s and task: %s."
+                        msg += WM_JOB_ERROR_CODES[job["failedErrCode"]]
+                        msg += " Marking it as FailedOnCreation"
+                        logging.error(msg, job['id'], job['jobgroup'], job['task'] )
```

Comment:
Given that this is an expected case, I would suggest changing the log level to warning instead of error.